### PR TITLE
Fix compatibility with dask 0.17.4

### PR DESCRIPTION
**Pull request by ericpre**
Tests are failing since last dask release (0.17.4). See PR #https://github.com/dask/dask/pull/3429 for the detailed changes.

### Progress of the PR
- [x] builtins has been removed from dask.numpy_compat: use builtins `abs` instead of calling it throught dask,
- [x] dask bumped numpy version to 1.11 so we need to do the same,
- [x] ready for review.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,7 +47,7 @@
 
 install_req = ['scipy>=0.15',
                'matplotlib>=1.2, !=2.1.0, !=2.1.1',
-               'numpy>=1.10, !=1.13.0',
+               'numpy>=1.11, !=1.13.0',
```
Comment (francisco-dlp) [inline setup.py]: Why is this needed? It's ``dask`` who requires v1.11 not HyperSpy.

**Comment by francisco-dlp** [comment]
Why not sending this to  Rnp?

File: setup.py (hunk 1 above)
Comment (ericpre) [reply setup.py]: Yes, indeed!

**Comment by ericpre** [comment]
Since the next release is going to be minor, I was thinking it would more simple to work on one branch only and to avoid merging next patch to next minor regularly? If you think it is still worth sticking to the normal workflow I will rebase on the patch branch.

**Comment by francisco-dlp** [comment]
Releases not always goes to plan e.g. we may find fix a serious issue before v1.4 is ready and decide to make a patch release, so I think that it'll be better resubmit to Rnp.
